tap_copper/streams/base.py

```diff
@@ -78,8 +78,10 @@ def save_state(self, last_record):
             save_state(self.state)
 
     def get_stream_data(self, response):
-
         transformed = []
+        if isinstance(response, dict):
+            ## flattens dict to list representation
+            response = [record for records in response.values() for record in records]
```

Comment:
Since this conditional is supposed to only affect the ActivityTypeStream, I'd suggest simply overriding the `get_stream_data` method in that class. Adding this change in the base class could impact existing streams unintentionally.